referentiel_evaluation.md

```diff
@@ -395,7 +404,7 @@ _(Domaine de risque spécifique : discrimination à l'encontre de certains group
 <details>
 <summary>Expl2.4 :</summary>
 
-Il s'agit de s'interroger systématiquement, à chaque projet de data science et selon l'objectif et l'usage cible du modèle que l'on veut élaborer, sur les features pouvant directement ou indirectement être à l'origine d'un risque de biais discriminatoire. On parle d'attribut protégé (*protected attribute* ou *protected variable* en anglais) pour désigner les attributs dont les valeurs définissent des sous-populations à risque de discrimination.
+Il s'agit de s'interroger systématiquement, à chaque projet de data science et selon l'objectif et l'usage cible du modèle que l'on veut élaborer, sur les features pouvant directement ou indirectement être à l'origine d'un risque de biais populationnel voire de discrimination. On parle d'attribut protégé (*protected attribute* ou *protected variable* en anglais) pour désigner les attributs dont les valeurs définissent des sous-populations à risque de biais et discrimination.
```

Comment:
Il manque selon moi une notion d'évolution dans le temps. Sur les updates de modèles, ainsi que des datasets de test / train, les distributions & risques de biais peuvent évoluer et doivent être ré-évalués avant de produire une nouvelle version.
Besmira Nushi propose de calculer des scores de compatibilité ascendante, pour vérifier que le traitement d'une sous-population reste robuste dans le temps.

Quelques références : 
- https://besmiranushi.com/docs/The%20Utopia%20of%20Human-AI%20Collaboration.pdf (slides 13->17)
- Nushi, B., Kamar, E., & Horvitz, E. (2018, June). Towards accountable ai: Hybrid human-machine analyses for characterizing system failure. In Proceedings of the AAAI Conference on Human Computation and Crowdsourcing (Vol. 6, No. 1)
- Une méthodo de surveillance & mise à jour de surveillance que j'ai décrit par ici : https://npublications.com/journals/educationinformation/2022/a102008-005(2022).pdf

Reply:
Tu as raison, c'est plus clair comme ça.